cartographer_grpc/framework/server.h

```diff
@@ -38,7 +39,8 @@ class Server {
   // All options that configure server behaviour such as number of threads,
   // ports etc.
   struct Options {
-    size_t number_of_threads = 4;
+    size_t num_grpc_threads = 2;
```

Comment:
Do we need these defaults? It feels like duplicating with the lua config.

Reply:
Done.